### DISCUSSION TITLE
Close picker on date selection

### DISCUSSION THREAD

**Discussion by johncarter-** [filamentphp/filament#3060] [Ideas]
Would a PR be accepted to allow devs to optionally close a `DatePicker`'s picker after a selection has been made?

```php
DatePicker::make('selectedEndDate')
    ->closeOnSelect()
```

![chrome_Ifyv4qJIAi](https://user-images.githubusercontent.com/3776888/177967106-6bea5898-5573-4e7e-a8a3-c2a2377b4147.gif)

**Comment by johncarter-** [comment]
For my own reference:  https://github.com/filamentphp/filament/blob/f9ae160ac876d38f8f435bb7ed82eca6b515fd62/packages/forms/resources/views/components/date-time-picker.blade.php#L150

**Comment by danharrin** [comment]
Yup! `closeOnDateSelection()`
